packages/docs/.remarkrc

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": [
+    "preset-lint-recommended",
+    "preset-lint-consistent",
+
```

Comment:
I would also recommend this great plugin: https://github.com/remarkjs/remark-lint

Reply:
It is already included: https://github.com/remarkjs/remark-lint/blob/master/packages/remark-preset-lint-consistent/index.js#L10

Reply:
Ah, my bad! 🤦‍♀